packages/demo/src/components/examples/MultiSelectHOCExamples.tsx

```diff
@@ -0,0 +1,55 @@
+import {locales} from 'moment';
+import * as React from 'react';
+import {
+    IItemBoxProps,
+    LabeledInput,
+    MultiSelectConnected,
+    UUID,
+    ValidationMessage,
+    withNonEmptyMultiSelectHOC,
+    withValidationMessage,
+} from 'react-vapor';
+import * as _ from 'underscore';
+
+const defaultItems: IItemBoxProps[] = [
+    {displayValue: 'Test', value: '0'},
+    {displayValue: 'One', value: '1'},
+    {displayValue: 'Disabled', value: 'disabled', disabled: true},
+    {displayValue: 'Three', value: '3'},
+    {displayValue: 'Four', value: '4'},
+    {displayValue: 'Five', value: '5'},
+    {displayValue: 'Six', value: '6'},
+    {displayValue: 'Seven', value: '7'},
+];
+
+const WithNonEmptyMultiSelect = withNonEmptyMultiSelectHOC(withValidationMessage(MultiSelectConnected));
+
+export interface IMultiSelectExamplesState {
+    first: IItemBoxProps[];
+    id: string;
+}
+
+export class MultiSelectHOCExamples extends React.Component<{}, IMultiSelectExamplesState> {
```

Comment:
🤔 I don't think this example brings much to the table compared to what is already `MultiSelectExamples` 

Why did you think we needed another section?

Any opinion from the admin-ui team on this?

Reply:
I used it to test out an example on the side (it was meant to be deleted 😅 )